package.json

```diff
@@ -68,14 +53,13 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-promise": "^6.0.0",
     "husky": "^4.3.8",
+    "jest": "^27.5.1",
     "lint-staged": "^10.5.4",
-    "mocha": "^8.4.0",
-    "mockery": "^2.1.0",
-    "nyc": "^15.1.0",
     "prettier": "^2.6.0",
     "serverless": "^3.7.8",
     "sinon": "^13.0.1",
```

Comment:
I think `sinon` can also be removed?

Reply:
No, I'm still using it to mock `process.cwd`